README.md

```diff
@@ -1,2 +1,23 @@
-# librosa_gallery
-A collection of example notebooks demonstrating librosa functionality
+# Librosa Gallery
+
+A collection of code examples demonstrating some of librosa's functionality.
+
+## Examples
+
+
+## Notebooks
+
+You can explore our Jupyter Notebooks to find out what functionality `librosa` offers you.
+As some example contain audio examples, you can use the `nbviewer` to see the notebooks.
+
+Use these links to get directly to the notebooks:
+* Intro: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/01-Intro_demo.ipynb
+* Audio Effects: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/02-Audio_effects.ipynb
+* HPRSS: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/02-HPRSS.ipynb
+* Vocal Separation: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/03-Vocal_separation.ipynb
+* Music Segmentation: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/04-Laplacian segmentation.ipynb
+* Superflux Onsets: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/05-Superflux_Onsets.ipynb
+* Rhythm Analysis: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/06-Rhythm_Analysis_Mellin_Transform.ipynb
+* Enhanced Chroma: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/07-Enhanced_Chroma.ipynb
+* Overriding Presets: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/08-Overriding_Defaults_with_Presets.ipynb
+* Music Synchronization: https://nbviewer.jupyter.org/github/librosa/librosa_gallery/blob/master/notebooks/09-Music_Synchronization.ipynb
```

Comment:
Why the file renames?  I'm fine with it, but wondering if it's just to make it easier for markdown links?